server/meshmodel/aws-prometheusservice-controller/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,55 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metdata": {
+    "description": "Hierarchical"
```

Comment:
"description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."